src/List.test.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import List from './List';
+
+describe('List', () => {
+  function getListComponentContainer(tasks) {
+    const { container } = render((
+      <List tasks={tasks} />
+    ));
+    return container;
```

Comment:
함수 내에서 바로 리턴해도 좋을 것 같아요~
```suggestion
    return render((
      <List tasks={tasks} />
    ));
```